include/materials/PolarPFMBetaS0.h

```diff
@@ -0,0 +1,29 @@
+/**********************************************************************/
+/*                     DO NOT MODIFY THIS HEADER                      */
+/* MAGPIE - Mesoscale Atomistic Glue Program for Integrated Execution */
+/*                                                                    */
+/*            Copyright 2017 Battelle Energy Alliance, LLC            */
+/*                        ALL RIGHTS RESERVED                         */
+/**********************************************************************/
+
+#pragma once
+
+#include "DerivativeParsedMaterialHelper.h"
+#include "ExpressionBuilder.h"
+
+/**
+ * Solid-melt gradient energy coefficient, eq. (7) from Physical Review B 89, 184102 (2014)
+ */
+class PolarPFMBetaS0 : public DerivativeParsedMaterialHelper, public ExpressionBuilder
+{
+public:
+  static InputParameters validParams();
+
+  PolarPFMBetaS0(const InputParameters & parameters);
+
+protected:
+  EBTerm _theta;
```

Comment:
What's the purpose of using the `ExpressionBuilder` for `_theta`?

Reply:
It is holding a symbolic reference to the theta variable. ExpressionBuilder is used to build the symbolic derivatives of the expression this material is representing.